pkg/restore/split_test.go

```diff
@@ -316,6 +319,36 @@ FindRegion:
 	return true
 }
 
+func (s *testRestoreUtilSuite) TestGetSplitKeys(c *C) {
```

Comment:
By removing `truncateRowKey`, does this test fail?

Reply:
This test only work on release-4.0...